src/main/resources/application.yaml

```diff
@@ -70,11 +70,22 @@ service-config:
       allow-creating-case-before-payments-are-processed: true
       form-type-to-surname-ocr-field-mappings:
         - formType: PERSONAL
-          ocrField: last_name
+          ocrFieldList:
+            - lastname_not_found
```

Comment:
what is this `lastname_not_found` for? this field doesn't exist in bulkscan ccd definition

Reply:
to test to see if is not found it takes the second one.

Reply:
it is added in integration tests so, it doesn't need to be here

Reply:
removed